testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4032,6 +4032,25 @@ function getColsCountFromDOM($form) {
     return result;
 }
 
+QUnit.test('Setting "colCountByScreen" property of group with dot notation should update layout', function(assert) {
+    const $form = $('#form').dxForm({
+        screenByWidth: () => 'md',
+        items: [
+            {
+                itemType: 'group',
+                colCountByScreen: { md: 3 },
+                items: [{ dataField: 'name' }, { dataField: 'email' }, { dataField: 'age' }]
+            }
+        ],
+    });
+
+    const instance = $form.dxForm('instance');
+
+    instance.option('items[0].colCountByScreen.md', 2);
```

Comment:
For more complecated path it works as well? For example
```
instance.option('items[0].item[1].colCountByScreen.md', 2);
or
form.itemOption('groupName.itemName', 'colCountByScreen', { md: '2' })
// https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxForm/Methods/#itemOptionid_option_value

Reply:
i didn't check  the case with 'itemOption' - thanks